helpers.go

```diff
@@ -324,3 +324,52 @@ func waitForEvent(emitter EventEmitter, event string, predicate ...interface{})
 	emitter.On(event, handler)
 	return evChan
 }
+
+type SelectOptionValues struct {
+	Value      *[]string
+	Index      *[]int
+	Label      *[]string
+	ElemHandle *[]ElementHandle
+}
+
+func ConvertSelectOptionSet(values SelectOptionValues) map[string]interface{} {
```

Comment:
I would not expose this method publicly (make it lowercase) and test it in the root directory like I do with the `helpers_test.go`.

Reply:
As `helpers_test.go` is in different package, `playwright_test`. If this function is not exposed I don't think it can be accessed in test. Or if implementation like SelectOptionValues.Convert()?

Reply:
`/helpers_test.go` has the `playwright` package so it should work. Generally I'd like to test external APIs in the tests folder and internal APIs in the root directory.